documentation/modules/exploit/windows/ssh/freesshd_authbypass.md

```diff
@@ -0,0 +1,92 @@
+## Description
+
+This module exploits a vulnerability found in FreeSSHd <= 1.2.6 to bypass authentication. You just need the username (which defaults to root). The exploit has been tested with both password and public key authentication.
+
+
+## Verification
+
+  1. Start msfconsole
+  2. Do : `use exploit/windows/ssh/freesshd_authbypass`
+  3. Do : `set RHOST [target IP]`
+  5. Do : `set PAYLOAD [valid windows payload]` if you want to use other payloads (`windows/meterpreter/reverse_tcp` by default)

```

Comment:
https://www.youtube.com/watch?v=VOaZbaPzdsk

Reply:
I have no excuse but sleeplessness and coffee highs.